### ISSUE TITLE
训练模型

### ISSUE THREAD

**Issue by Znc1996** [csuhan/ReDet#33]
我在训练模型时总是报有这样的错误，但是最后也可以训练出结果，可以拿去测试，请问这个是怎么回事呀？

2021-06-23 09:19:23,956 - WARNING - The model and loaded state dict do not match exactly

unexpected key in source state_dict: backbone.conv1.weights, backbone.conv1._basisexpansion.block_expansion_('irrep_0', 'regular').sampled_basis, backbone.bn1.indices_8, backbone.bn1.batch_norm_[8].weight, backbone.bn1.batch_norm_[8].bias, backbone.bn1.batch_norm_[8].running_mean, backbone.bn1.batch_norm_[8].running_var, backbone.bn1.batch_norm_[8].num_batches_tracked, backbone.layer1.0.conv1.weights, backbone.layer1.0.conv1.filter, backbone.layer1.0.conv1._basisexpansion.block_expansion_('regular', 'regular').sampled_basis, backbone.layer1.0.bn1.indices_8, backbone.layer1.0.bn1.batch_norm_[8].weight, backbone.layer1.0.bn1.batch_norm_[8].bias, backbone.layer1.0.bn1.batch_norm_[8].running_mean, backbone.layer1.0.bn1.batch_norm_[8].running_var, backbone.layer1.0.bn1.batch_norm_[8].num_batches_tracked, backbone.layer1.0.conv2.weights, backbone.layer1.0.conv2.filter, backbone.layer1.0.conv2._basisexpansion.block_expansion_('regular', 'regular').sampled_basis, backbone.layer1.0.bn2.indices_8, backbone.layer1.0.bn2.batch_norm_[8].weight, backbone.layer1.0.bn2.batch_norm_[8].bias, backbone.layer1.0.bn2.batch_norm_[8].running_mean, backbone.layer1.0.bn2.batch_norm_[8].running_var, backbone.layer1.0.bn2.batch_norm_[8].num_batches_tracked, backbone.layer1.0.conv3.weights, backbone.layer1.0.conv3.filter, backbone.layer1.0.conv3._basisexpansion.block_expansion_('regular', 'regular').sampled_basis, backbone.layer1.0.bn3.indices_8, backbone.layer1.0.bn3.batch_norm_[8].weight, backbone.layer1.0.bn3.batch_norm_[8].bias, backbone.layer1.0.bn3.batch_norm_[8].running_mean, backbone.layer1.0.bn3.batch_norm_[8].running_var, backbone.layer1.0.bn3.batch_norm_[8].num_batches_tracked, backbone.layer1.0.downsample.0.weights, backbone.layer1.0.downsample.0.filter, backbone.layer1.0.downsample.0._basisexpansion.block_expansion_('regular', 'regular').sampled_basis, backbone.layer1.0.downsample.1.indices_8, backbone.layer1.0.downsample.1.batch_norm_[8].weight, backbone.layer1.0.downsample.1.batch_norm_[8].bias, backbone.layer1.0.downsample.1.batch_norm_[8].running_mean, backbone.layer1.0.downsample.1.batch_norm_[8].running_var, backbone.layer1.0.downsample.1.batch_norm_[8].num_batches_tracked, backbone.layer1.1.conv1.weights, backbone.layer1.1.conv1.filter, backbone.layer1.1.conv1._basisexpansion.block_expansion_('regular', 'regular').sampled_basis, backbone.layer1.1.bn1.indices_8, backbone.layer1.1.bn1.batch_norm_[8].weight, backbone.layer1.1.bn1.batch_norm_[8].bias, backbone.layer1.1.bn1.batch_norm_[8].running_mean, backbone.layer1.1.bn1.batch_norm_

感谢您

**Comment by csuhan** [comment]
ReResNet预训练权重没有正确加载

**Comment by Znc1996** [comment]
re_resnet50_c8_batch256-12933bc2.pth请问您有这个预训练权重的链接吗？

**Comment by csuhan** [comment]
See [https://github.com/csuhan/ReDet/blob/master/README.md]()

<html>
<body>
<ul style="box-sizing: border-box; padding-left: 2em; margin-top: 0px; margin-bottom: 16px; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><li style="box-sizing: border-box;"><strong style="box-sizing: border-box; font-weight: 600;">ImageNet pretrain</strong></li></ul><p style="box-sizing: border-box; margin-top: 0px; margin-bottom: 16px; color: rgb(36, 41, 46); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">We pretrain our ReResNet on the ImageNet-1K. Related codes can be found at the<span> </span><a href="https://github.com/csuhan/ReDet/tree/ReDet_mmcls" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none;">ReDet_mmcls</a><span> </span>branch. Here we provide our pretrained ReResNet-50 model for convenience. If you want to train and use ReResNet in your own project, please check out<span> </span><a href="https://github.com/csuhan/ReDet/tree/ReDet_mmcls" style="box-sizing: border-box; background-color: transparent; color: var(--color-text-link); text-decoration: none;">ReDet_mmcls</a><span> </span>for the installation and basic usage.</p>

Model | Group | Top-1 (%) | Top-5 (%) | Download
-- | -- | -- | -- | --
ReR50 | C8 | 71.20 | 90.28 | model \| log


</body>
</html>
